pages/api/csp-report.ts

```diff
@@ -1,11 +1,10 @@
 import { NextApiRequest, NextApiResponse } from 'next';
-import { serverLogger } from 'features/loggers';
 
 export default function cspReport(
   req: NextApiRequest,
   res: NextApiResponse,
 ): void {
-  serverLogger.warn({
+  console.warn({
```

Comment:
console will write to stderr, you sure we're currently writing to it?

Reply:
Yep, it will be properly handled by pino + next logger (you have probably seen log level: `warn` and `error` before, e.g. yesterday)